index.php

```diff
@@ -9,4 +9,135 @@
    Author URI: https://mxstbr.com
    License: MIT
    */
+
+// Inject the feedback.fish JavaScript file into the DOM just before the body
+function enqueue_feedback_fish_script()
+{
+    $projectId = get_option('feedback_fish_project_id');
+    wp_register_script(
+        'feedback-fish',
+        "https://feedback.fish/ff.js?pid=$projectId",
+        [],
+        null,
+        true
+    );
+    wp_enqueue_script('feedback-fish');
+}
+add_action('wp_enqueue_scripts', 'enqueue_feedback_fish_script');
+
+// Add defer="defer" to the feedback.fish <script> tag to avoid blocking render
+// Taken from: https://wordpress.stackexchange.com/a/38335
+add_filter(
+    'script_loader_tag',
+    function ($tag, $handle) {
+        if ('feedback-fish' !== $handle) {
+            return $tag;
+        }
+
+        return str_replace(' src', ' defer="defer" src', $tag);
+    },
+    10,
+    2
+);
+
+// Add "Send feedback" to the primary nav if it exists
+function add_feedback_fish_nav_menu_item($items, $args)
+{
+    $current_user = wp_get_current_user();
+    $manual_usage = get_option('feedback_fish_manual');
```

Comment:
Since users can have many menus used in many different places, I'd suggest having it be opt-in to specific menu(s) instead of automatically on for all menus and opt-out. 

On the settings page, instead of an option to "I will add the HTML attribute myself" checkbox, you could provide a dropdown of existing menus, and let the user select which Menu to assign feedback fish to. 

Then instead of `if ( ! $manual_usage ) {`:

You could do something like:

```php
// Get the menu that was assigned
$ff_assigned_menu_id = get_option( 'feedback_fish_assigned_menu', 0 );

// If the assigned menu id matches the menu being filtered
if ( ! empty( $ff_menu_id ) && $args->menu === $ff_assigned_menu_id ) {
  // do your thing
}
```